lib/std/once.zig

```diff
@@ -51,15 +52,18 @@ test "Once executes its function just once" {
         global_once.call();
     } else {
         var threads: [10]std.Thread = undefined;
-        defer for (threads) |handle| handle.join();
+        var thread_count: usize = 0;
+        defer for (threads[0..thread_count]) |handle| handle.join();
```

Comment:
Not sure what's the idiomatic pattern for array deinit deferral!

Reply:
Not sure there is one. This is perfectly fine.